iroh-resolver/src/unixfs.rs

```diff
@@ -0,0 +1,97 @@
+use anyhow::{anyhow, bail, Result};
+use bytes::{Buf, Bytes};
+use cid::Cid;
+use prost::Message;
+
+mod unixfs_pb {
+    include!(concat!(env!("OUT_DIR"), "/unixfs_pb.rs"));
+}
+
+mod dag_pb {
+    include!(concat!(env!("OUT_DIR"), "/merkledag_pb.rs"));
```

Comment:
same as above, possibly replace with
`tonic::include_proto!("merkledag_pb");`

Reply:
no, tonic is doing grpc, this is raw protobufs, so we don’t want to use tonic but prost directly